google-cloud-language/google-cloud-language.gemspec

```diff
@@ -9,22 +9,22 @@ Gem::Specification.new do |gem|
 
   gem.authors       = ["Google LLC"]
   gem.email         = "googleapis-packages@google.com"
-  gem.description   = "google-cloud-language is the official library for Cloud Natural Language API."
-  gem.summary       = "API Client library for Cloud Natural Language API"
-  gem.homepage      = "https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-language"
+  gem.description   = "Provides natural language understanding technologies, such as sentiment analysis, entity recognition, entity sentiment analysis, and other text annotations."
```

Comment:
Where was this description obtained? "natural language understanding technologies" seems a little off. Should it be replaced with something taken directly from the [official product documentation](https://cloud.google.com/natural-language), such as "Natural Language uses machine learning to reveal the structure and meaning of text."

Reply:
The text comes from https://cloud.google.com/natural-language/docs